docs/python/building.py

```diff
@@ -0,0 +1,71 @@
+class Building(object):
+	"""Class representing all the data for a building
+	
+	'attribute name': 'type'
+	swagger_types = {
+		'status': 'str',
+		'id': 'str',
+		'nameList': 'list[str]',
+		'numWashers': 'int',
+		'numDryers': 'int',
+	}
+	
+	'attribute name': 'Attribute name in Swagger Docs'
+	attribute_map = {
+		'status': 'status',
+		'id': 'id',
+		'nameList': 'nameList',
+		'numWashers': 'numWashers',
+		'numDryers': 'numDryers'
+	}
+	"""
+	
+	def __init__(self, status, ID, nameList, numWashers, numDryers):
+		"""Class instatiation
+		
+		Check if all the attributes are valid and assigns them if they are
+		Raises ValueError if attributes are invalid
+		"""
+		allowed_status = ["success", "failure"]
+		if status not in allowed_status:
+			raise ValueError(
+					"Invalid value for 'status' ({0}), must be one of {1}"
+					.format(status, allowed_status))
+		
+		if ID is None:
+			raise ValueError("Invalid value for 'ID', must not be 'None'")
+		
+		if nameList is None:
+			raise ValueError("Invalid value for 'nameList', must not be 'None'")
+		
+		if numWashers is None:
+			raise ValueError("Invalid value for 'numWashers', must not be 'None'")
+		
+		if numDryers is None:
+			raise ValueError("Invalid value for 'numDryers', must not be'None'")
+		
+		self.status = status
+		self.ID = ID
+		self.nameList = nameList
+		self.numWashers = numWashers
+		self.numDryers = numDryers
+		
+	def status(self):
```

Comment:
These functions aren't needed. Users will just do `building.status` to get the status, since it would do the same thing as `building.status()` but without the parentheses.